app/src/main/java/com/example/tung/moviedb_07/data/model/ProductionCountry.java

```diff
@@ -2,37 +2,32 @@
 
 import android.os.Parcel;
 import android.os.Parcelable;
-import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
-/**
- * Created by tung on 6/8/17.
- */
-
 public class ProductionCountry implements Parcelable {
 
     @SerializedName("iso_3166_1")
-    @Expose
     private String mIso31661;
-
     @SerializedName("name")
-    @Expose
     private String mName;
 
-    public static final Creator<ProductionCountry> CREATOR = new Creator<ProductionCountry>() {
-
-        @SuppressWarnings({ "unchecked" })
-        public ProductionCountry createFromParcel(Parcel in) {
-            ProductionCountry instance = new ProductionCountry();
-            instance.mIso31661 = ((String) in.readValue((String.class.getClassLoader())));
-            instance.mName = ((String) in.readValue((String.class.getClassLoader())));
-            return instance;
-        }
-
-        public ProductionCountry[] newArray(int size) {
-            return (new ProductionCountry[size]);
-        }
-    };
+    public final static Creator<ProductionCountry> CREATOR =
+            new Creator<ProductionCountry>() {
+
+                @SuppressWarnings({
+                        "unchecked"
+                })
+                public ProductionCountry createFromParcel(Parcel in) {
+                    ProductionCountry instance = new ProductionCountry();
+                    instance.mIso31661 = ((String) in.readValue((String.class.getClassLoader())));
+                    instance.mName = ((String) in.readValue((String.class.getClassLoader())));
+                    return instance;
```

Comment:
ERROR: 'method def' child have incorrect indentation level 20, expected level should be one of the following: 12, 16.